app/main.ipynb

```diff
@@ -212,7 +212,6 @@
     ")\n",
     "top_by_money.plot.pie(\n",
     "    y=\"total_price\",\n",
-    "    labels=top_by_money.name,\n",
```

Comment:
My bad! Everything fine with lables, you should remove the legend to not provide information about colors of diagram

Reply:
all good :o)  cheers for clarifying